buildSrc/src/main/kotlin/LicenseUtils.kt

```diff
@@ -34,6 +34,7 @@ object CopyrightableFiles {
         "plugins/reporters/asciidoc/src/main/resources/pdf-theme/pdf-theme.yml",
         "plugins/reporters/asciidoc/src/main/resources/templates/freemarker_implicit.ftl",
         "plugins/reporters/fossid/src/main/resources/templates/freemarker_implicit.ftl",
+        "plugins/reporters/scanoss/src/main/resources/templates/freemarker_implicit.ftl",
```

Comment:
Can we make a cut here and only keep the preceding 5 commits in this PR, and create separate PRs for the next 4 commits? (Probably 2 or 3 other PRs that each focus on report generation or refactoring.)

Reply:
> Would you prefer create all three PRs at once and mark the dependent ones as "Draft" until their predecessors are merged, or should I submit them one at a time as each gets approved?

Either way is fine with me. Maybe the latter creates a bit less "clutter" due to open draft PRs.

> I can prepare these separate PRs right away once we agree on the approach.

Great, thanks. Please do ahead as you see fit.

Reply:
First PR is #10265

Reply:
@isasmendiagus, is it ok to close this PR already to clean things up, as all its remainders are now included in https://github.com/oss-review-toolkit/ort/pull/10287?